### ISSUE TITLE
definition of an unknown language item: panic_fmt

### ISSUE THREAD

**Issue by fmmrtns** [japaric/stm32f103xx-hal#88]
Hi! Thanks for this crate! 

I'm having a problem when building `cortex-m-quickstart` which adds this crate as a dependency:
```
error[E0522]: definition of an unknown language item: `panic_fmt`
--> .../.cargo/registry/src/github.com-1ecc6299db9ec823/panic-semihosting-0.2.0/src/lib.rs:67:1
    |
    | #[lang = "panic_fmt"]
    | ^^^^^^^^^^^^^^^^^^^^^ definition of unknown language item `panic_fmt`
```

Bumping `panic-semihosting` to version 0.3.0 should fix the issue, but I'm not aware of potential side-effects of this change in this create. Is there anything else I can do to fix this?

Thanks

**Comment by fmmrtns** [comment]
Just saw this PR which fixes this issue: https://github.com/japaric/stm32f103xx-hal/pull/86 :+1:

**Comment by japaric** [comment]
Fixed in #86
